backend/lib/fetch/providers.py

```diff
@@ -125,14 +120,12 @@ def geocode_providers(providers, geocoder_name=GEOCODER, engine=connect.create_d
     elif not GEOCODING:
         logger.debug('Warning - Geocoding is not active. Processing without missing addresses.')
 
-    for i, raw_provider in enumerate(providers):
+    for i, raw_address in enumerate(providers):
```

Comment:
Still a little odd to me that the elements of the list are called `raw_address` but the list is called `providers`.

Reply:
Switched